src/main/java/com/epam/jconference/controller/EventController.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.jconference.controller;
+
+import com.epam.jconference.controller.dto.EventDto;
+import com.epam.jconference.controller.dto.EventFilterDto;
+import com.epam.jconference.service.model.Event;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+public class EventController {
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping("/events")
+    public EventDto create(@RequestBody EventDto eventDto) {
+        return null;
+    }
+
+    @GetMapping("/events")
```

Comment:
If all you methods has the same url, like /events you can use @RequestMapping("/events") before the class name, and just use @GetMapping("{id}") for example, instead of duplicate "/events" before each one.